Cargo.toml

```diff
@@ -26,6 +26,10 @@ path = "src/format-diff/main.rs"
 name = "git-rustfmt"
 path = "src/git-rustfmt/main.rs"
 
+[[bin]]
+name = "check_diff"
+path = "ci/check_diff/main.rs"
+
```

Comment:
I think it would be better to make this its own crate instead of defining it as another binary target in rustfmt.